.github/workflows/replay-verify.yaml

```diff
@@ -90,7 +90,7 @@ jobs:
       SUB_DIR: e1
       HISTORY_START: 518000000
       #TXNS_TO_SKIP: 12253479 12277499 148358668
-      TXNS_TO_SKIP: "0"
+      TXNS_TO_SKIP: 523296049 523298111 575378008 575383660 575455170 575457845 575461986 575470789 596888095 612286393 981890562 1011802725 1030023658 1652667187 1652669110 1652686860 1652689144 1652722847 1654190659 1654191460 1673664597 1730942524 1730944006 1730945331 1730945814 1730946863 1730952284 1730956071 1730959853 1730967212 1731006027 1759904543 1792070645 1797308520 1798657294 1806178096
```

Comment:
@sherry-x are we fine with skipping transactions on mainnet?

@georgemitenkov how recent are these transactions? If leader v2 is onchain flag - what do these represent?

Reply:
For this range, the first skip is 27.03.24 and the last is basically now: 17.10.24. The reason why we want to skip is that we can completely remove old V1 code then, and the trade off few dozen of transactions that publish code seems good.

See the spreadsheet for actual divergences, those are only due to bugs in old loader (went to through them one by one). For instance, we were flushing module cache and losing error messages, there are a number of cases where the difference is None vs Some(msg) in VMStatus. There are also 2 other cases: 1) init_module not linking correctly before, 2) not topologically sorted bundles not linking correctly and failing to publish.

Ideally, I would not skip but specify the difference in write set, but we cannot do it with the current replay, hence skipping and documenting in a spreadsheet what is the diff.